### ISSUE TITLE
Create Rename All

### ISSUE THREAD

**Issue by marinofaggiana** [nextcloud/ios#2118]
Create a rename form for rename more file with a mask

![IMG_8307](https://user-images.githubusercontent.com/8616947/184892392-0857c2d5-4515-4346-a383-de078aff4b62.jpg)

**Comment by tobiasKaminsky** [comment]
@jancborchardt your idea? As we do not have this on sever, nor on any other client.
PC also do not have this, but rely on 3rd party apps for bulk renaming.

**Comment by jancborchardt** [comment]
Since we currently don’t have this anywhere, I’d recommend to first discuss in an overview if we want to have it. Otherwise we move away from feature parity among our own apps. :)

**Comment by tobiasKaminsky** [comment]
@jancborchardt 👍

Where shall we discuss this? In server repo, as this would then also be there, plus in iOS/Android later then.

@marinofaggiana be aware that with webdav clients always have to rename files one by one. There is no endpoint to rename e.g. 10 files at once.

**Comment by marinofaggiana** [comment]
@tobiasKaminsky yes but this is not a problem, 20 files renamed with a.g. MAX 10 parallel task are are almost instantaneous
